cfgov/v1/atomic_elements/organisms.py

```diff
@@ -281,32 +284,53 @@ class Media:
         js = ['bureau-structure.js']
 
 
-class AtomicTableInput(TableInput):
+class RichTextTableInput(WidgetWithScript, forms.HiddenInput):
```

Comment:
@willbarton mainly because previously we weren't really using `TableInput` ([there's not much more to it](https://github.com/wagtail/wagtail/blob/master/wagtail/contrib/table_block/blocks.py#L14)), and skipping its `render` in favor of its grandparent's anyway. This more explicitly defines our own widget that uses rich text.